src/helpers/reddit_client.ts

```diff
@@ -56,4 +75,56 @@ export class RedditClient {
             return [];
         }
     }
+
+    async getTopDayPosts(): Promise<Array<KpopNewsRedditPost>> {
+        try {
+            const matchingPosts = await this.client.search({
+                subreddit: "kpop",
+                query: generateFilteredQuery(),
+                sort: "top",
+                time: "day",
+            });
+
+            const popularPosts = matchingPosts.filter((x) => x.score > 100);
```

Comment:
Is karma > 100 too high for daily posts?

Reply:
Also, do we want to limit the number of posts?

Reply:
Currently, there are 20 posts above 100 in the past 24 hours. I can set a limit to 25, which is the first page of r/kpop on old reddit